jmclient/jmclient/auth.py

```diff
@@ -0,0 +1,94 @@
+import datetime
+import os
+from typing import Optional
+
+import jwt
+
+from jmbase.support import bintohex
+
+
+class InvalidScopeError(Exception):
+    pass
+
+
+class JMSessionValidity:
+    access = datetime.timedelta(minutes=30)
+    refresh = datetime.timedelta(hours=4)
```

Comment:
Nit: Questionable whether there's any point in a class here, if we just have two hardcoded values.

Reply:
Questionable indeed, I did wonder myself and ended up using a class for he following reasons.

1. Both values relates to a session validity, where each cannot exits without the other. It is the same object imho.
2. It allows to programmatically access one or the other value with `getattr(self.validity, token_type)`
3. It is consistent with `JMTokenAuthority.signature` attribute